app/Domain/Integrations/Repositories/IntegrationRepository.php

```diff
@@ -17,8 +18,8 @@ public function getById(UuidInterface $id): Integration;
     public function getByIdWithTrashed(UuidInterface $id): Integration;
     public function deleteById(UuidInterface $id): ?bool;
     public function getByContactEmail(string $email, ?string $searchQuery): PaginatedCollection;
-    public function requestActivation(UuidInterface $id, UuidInterface $organizationId, ?string $couponCode): void;
+    public function requestActivation(UuidInterface $id, UuidInterface $organizationId, ?string $couponCode, UdbOrganizerCollection $organizers=null): void;
     public function activate(UuidInterface $id): void;
-    public function activateWithOrganization(UuidInterface $id, UuidInterface $organizationId, ?string $couponCode): void;
+    public function activateWithOrganization(UuidInterface $integrationId, UuidInterface $organizationId, ?string $couponCode, array $organizers=[]): void;
```

Comment:
Why change a single parameter? Is it not better to keep it consistent with the rest of the methods?